pkgs/by-name/e-/e-imzo-manager/package.nix

```
@@ -0,0 +1,80 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  appstream,
+  appstream-glib,
+  cargo,
+  clippy,
+  desktop-file-utils,
+  gettext,
+  git,
+  meson,
+  ninja,
+  pkg-config,
+  polkit,
+  rustc,
+  rustPlatform,
+  wrapGAppsHook4,
+  gdk-pixbuf,
+  glib,
+  adwaita-icon-theme,
+  gtk4,
+  libadwaita,
+  openssl,
+}:
+stdenv.mkDerivation rec {
+  pname = "e-imzo-manager";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "xinux-org";
+    repo = "e-imzo";
+    rev = version;
+    hash = "sha256-uDaqkz2VDvqTgi+k8EGGKjLkjoH93xXHQcgUc1NVo30=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit pname version src;
+    hash = "sha256-rulWG4L/uN6+JBk+SzC0y57Pdw5N0Q1dJlpXGVo+vbQ=";
+  };
+
+  nativeBuildInputs = [
+    appstream
+    appstream-glib
+    cargo
+    clippy
+    gettext
+    git
+    meson
+    ninja
+    pkg-config
+    polkit
+    rustc
+    rustPlatform.cargoSetupHook
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [
+    desktop-file-utils
+    gdk-pixbuf
+    glib
+    adwaita-icon-theme
+    gtk4
+    libadwaita
+    openssl
+    rustPlatform.bindgenHook
+  ];
+
```

```suggestion
  passthru.updateScript = nix-update-script { };

```

Add `nix-update-script`

---

Resolved. Thank you very much for your time and awesome feedback!